plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPluginExtension.kt

```diff
@@ -56,4 +56,11 @@ abstract class SentryPluginExtension @Inject constructor(project: Project) {
      */
     val forceInstrumentDependencies: Property<Boolean> = objects.property(Boolean::class.java)
         .convention(false)
+
+    /**
+     * Enabled debug output of the plugin. Useful when there are issues with code instrumentation,
+     * shows the modified bytecode.
+     * Defaults to false.
+     */
+    val debug: Property<Boolean> = objects.property(Boolean::class.java).convention(false)
```

Comment:
As a side note, this is a `debug` option but only for code instrumentation. So `debugInstrumentation` (or similar) would be a better name.

Just `debug` gives the false feeling that you're enabling `debug` debugging for the whole plugin (which would be a nice feature but it's not implemented).

Reply:
what else do you think is missing if it was a `debug` for the whole plugin?

Reply:
since we already use `org.gradle.api.Logger` everywhere already and already has logging capabilities, I'd also go with `debugInstrumentation` and don't spend time implementing a file logging for the whole plugin